mobify/split-test
# Removes using dependencies in bower, removes dist, updates instructions

Status: **Opened for mergy mergy**

Reviewers: Any Mobify engineer
## Changes
- Remove bower
- Remove dist
- Update instructions to use npm
## How to test-drive this PR
- Follow npm install instructions, make sure you get the split-test library!
- Run `grunt test`, make sure tests pass
